src/NUnitFramework/nunitlite.tests/TextUIReportTests.cs

```diff
@@ -148,6 +149,19 @@ private IList<string> GetReportLines()
             return lines;
         }
 
+        private IList<string> RemoveInvokeStubLines(IList<string> lines)
+        {
+            var cleanLines = new List<string>(lines.Count);
+            foreach ( var line in lines )
+            {
+                if ( !line.Contains("InvokeStub_") )
```

Comment:
Thanks for catching this. It's odd that we can see stubs in the stack traces on .net 7. I wonder if this will go away by GA.

Reply:
I assume it is temporary, if not, it would be better to add the test in `GetReportLines` instead of creating a copy.

Reply:
Unfortunately, in the release version behaviour is the same, made proposed changes